docs/Change_Log.md

```diff
@@ -1,5 +1,46 @@
 # Change Log
 
+## API and Gateway V8
+
+#### September 24, 2020
+
+We've introduced API and Gateway v8! Changes are noted throughout the documentation, but the major changes are:
+
+- API v8 is now available. v6 is still the default for the time being.
+- All permissions have been converted to strings-serialized numbers. As such, `permissions_new`, `allow_new`, and `deny_new` have been removed
+- The `game` field has been removed. If you need a direct replacement, you can instead reference the first element of `activities`
+- Channel Permission Overwrite `type`s are now numbers (0 and 1) instead of strings ("role" and "member"). However due to a current technical constraint, they are string-serialized numbers in audit logs.
```

Comment:
```suggestion
- Channel Permission Overwrite `type`s are now numbers (0 and 1) instead of strings ("role" and "member"). However due to a current technical constraint, they are string-serialized numbers in audit log `options`.
```